src/utils/api.ts

```diff
@@ -157,18 +158,24 @@ export const changePassword = ({
 };
 
 export const getChatRooms = ({
+  chatRoomsSlug = 'all',
   limit,
   offset,
 }: {
+  chatRoomsSlug: string;
   limit: number | string;
   offset: number | string;
 }) => {
-  return api.get(`/dashboard/chat/rooms/?limit=${limit}&offset=${offset}`);
+  const url =
+    chatRoomsSlug === 'all'
+      ? `/dashboard/chat/rooms/?limit=${limit}&offset=${offset}`
+      : `/dashboard/chat/rooms/?inbox=${chatRoomsSlug}&limit=${limit}&offset=${offset}`;
+  return api.get(url);
 };
 
-export const getChatRoom = (_: string, userId: number) => {
-  return api.get(`/dashboard/chat/rooms/${userId}/`);
-};
+// export const getChatRoom = (_: string, userId: number) => {
+//   return api.get(`/dashboard/chat/rooms/${userId}/`);
+// };
 
```

Comment:
Why is this commented out? Maybe you can remove if its not needed or provide a reason why it might be useful

Reply:
It's no longer used, will remove it